src/document.h

```diff
@@ -1640,6 +1640,19 @@ struct Document {
                 tags.erase(c->text.t);
                 Refresh();
                 return nullptr;
+
+            case A_FILTERBYCELLBG: 
+                uint cellcolor = c->cellcolor;
+                loopallcells(ci) {
+                    if (ci->cellcolor != cellcolor) {
+                        ci->text.filtered = true;

```

Comment:
`ci->text.filtered = ci->cellcolor != c->cellcolor` ?

Reply:
This is cool! Thanks for the awesome suggestion! I revised the commit accordingly.